tests/tasks/spacy/test_spacy_tasks.py

```diff
@@ -17,6 +17,11 @@ def test_initialization(self):
         task = SpacyNLP(text="This is some text", nlp=spacy.blank("en"))
         assert task.text == "This is some text"
 
+    def test_load_nlp_model(self):
```

Comment:
Ideally, I would want this to run on both spacy<3.0 and spacy>3.0! 
So, i was wondering - is there a way to achieve this with CI? Or some other way?

Reply:
I don't think we can test against multiple versions simply here. I'd manually test against the old version and have CI test the latest for now.

Reply:
You can include a test that mocks the spacy version and asserts that the correct call is made for < 3.0

Reply:
> You can include a test that mocks the spacy version and asserts that the correct call is made for < 3.0

Ah, this makes sense, I'll do this, thanks!